index.html

```diff
@@ -48,7 +50,200 @@ <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
+          <div class="mastery">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+
+          <button class="btn btn-mastery" type="button">See Project</button>
+        </div>
+      </div>
+
+      <div class="title grid">
+        <div>
+          <img src="./images/Snapshoot_Portfolio2.png" alt="image"/>
+        </div>
+        <div class="title-text">
+          <div class="t-text">
+            <div class="port">
+              <h4>Tonic</h4>
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
+          <div class="mastery">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn btn-mastery" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="title grid">
+        <div>
+          <img src="./images/Snapshoot_Portfolio3.png" alt="portfolio"/>
+        </div>
+        <div class="title-text">
+          <div class="t-text">
+            <div class="port">
+              <h4>Multi-Post Stories</h4>
+
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <div class="q-text">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </div>
+          <div class="mastery">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+
+          <button class="btn btn-mastery" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+
+    <section class="about">
+      <div class="container">
+        <div>
+          <h2 class="about_header">About<br />Myself</h2>
+          <p class="win">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+        </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_60](https://user-images.githubusercontent.com/69638013/165106884-903b0694-9025-4826-b1ef-1dcc3c905cfd.png)  | ![Screenshot_61](https://user-images.githubusercontent.com/69638013/165108103-3f6cac6f-b28e-4b4a-adb8-1870f71fc891.png)

- [ ] Kindly adjust the white background color to fill the entire width of this section, so your design looks exactly like the Figma design